Makefile

```diff
@@ -3,32 +3,23 @@ TESTS=tests/*.js
 MONGO_CONNECTION?=mongodb://localhost/test_db
 CUSTOMCONNSTR_mongo_settings_collection?=test_settings
 CUSTOMCONNSTR_mongo_collection?=test_sgvs
-
-BLANKET=--require blanket 
+MONGO_SETTINGS=MONGO_CONNECTION=${MONGO_CONNECTION} \
```

Comment:
This looks sketchy but seems to work. I'm trying to keep the code dry since `test` and `travis-cov` use the same mongo settings.